src/client.ts

```diff
@@ -23,15 +22,18 @@ export interface SubscriptionOptions {
 }
 
 export interface Subscription {
-  options: SubscriptionOptions,
-  handler: (error: Error[], result?: any) => void,
+  options: SubscriptionOptions;
+  handler: (error: Error[], result?: any) => void;
 }
 
 export interface Subscriptions {
   [id: string]: Subscription;
 }
 
+export type ConnectionParams = {[paramName: string]: string};
```

Comment:
This type seems overly specific. Why should param values have to be strings?  (Does TypeScript have a JSONable type, or is `any` good enough?)